docs/getting_started/basics/nesting_workflows.md

```diff
@@ -1,16 +1,22 @@
 # Nesting Workflows
 
-In Shift any workflow can be added and executed as an operator.
+In Shift, any workflow can be embedded into a node and executed within another workflow.
 
 ## Setting Workflow I/Os
+To create a reusable workflow, the input and output of the workflow must be specified. This is done through `Input` and `Output` nodes. The plugs created in these nodes define which inputs are expected by the workflow and which are the outputs that will be returned.
```

Comment:
```suggestion
To create a reusable workflow, the inputs and outputs of the workflow must be specified. This is done through `Input` and `Output` nodes. The `through` plugs created in these nodes define which inputs are expected by the workflow and which are the outputs that will be returned.
```

Reply:
Should we link this concept somewhere? I feel it's a bit out of the blue to mention its a through plug

Reply:
I think we can mention it in this documentation as we mostly use through plugs for input and output operators

Reply:
I changed the sentence